### PR TITLE
YARN-11522. Update the documentation with the YARN-11000 changes.

### PR DESCRIPTION
**Pull request by tomicooler**
### For code changes:

- [x] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'YARN-11522. Your PR title ...')?
- [ ] Object storage: have the integration tests been executed and the endpoint declared according to the connector-specific documentation?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the `LICENSE`, `LICENSE-binary`, `NOTICE-binary` files?

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 30s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  markdownlint  |   0m  0s |  |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  33m 50s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 24s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  54m 50s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 13s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 16s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  20m 37s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 29s |  |  The patch does not generate ASF License warnings.  |
|  |   |  79m 10s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-5870/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/5870 |
| Optional Tests | dupname asflicense mvnsite codespell detsecrets markdownlint |
| uname | Linux 5d83ecd263a9 4.15.0-213-generic #224-Ubuntu SMP Mon Jun 19 13:30:12 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 96d1eb8684f7ca9d682449a4873702ddaba80f32 |
| Max. process+thread count | 564 (vs. ulimit of 5500) |
| modules | C: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site U: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-5870/1/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by slfan1989** [comment]
@tomicooler Thank you for your contribution! I quickly read it, LGTM. I will take a closer look later.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md
Comment (slfan1989) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md:2146]: Is this change necessary?

Comment (tomicooler) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md:2146]: Thanks for the review. Yes, it fixes fenced code block in the markdown, the `json` syntax highlighting hint needs to be added only at the start not at the end. (https://www.markdownguide.org/extended-syntax/#syntax-highlighting)

It looks like this: https://github.com/tomicooler/hadoop/blob/YARN-11522/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md

**Review by slfan1989 — APPROVED** [review]

**Comment by slfan1989** [comment]
@brumi1024 Can you double check this pr? Thank you very much!

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Nit: "combining absolute, weight or percentage" I would add values to the end, and drop the rest. For me in this form it sounds like this is an alternative to the percentage description, but it's a combination of all three before it.

This provides the most flexible way **of control for admins** to configure required amount of resources for a given queue. -> I don't think "of control for admins" is needed here, without it sound cleaner.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Nit: I would put the default part to the end.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Nit: cpu -> Vcores, memory -> Memory, gpu -> GPU.

gpu is set in absolute unit -> gpu is set in absolute units

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Effective min/max resources are an internal detail, with no mention of them anywhere on this page, so I think we should avoid introducing them in this manner. We could simply mention the hierarchy between the resources.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Same as above.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: YARN-11520 is under review, so we could wait for it with this update.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: effective absolute -> same as above, effective resources aren't introduced.

**Review by brumi1024 — COMMENTED** [review]
Thanks @tomicooler for the update! Had a few small comments.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md
Comment (tomicooler) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md:309]: Is it OK to use the `effective` term here? The json/xml responses are full of effective* objects anyway.

@brumi1024

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (tomicooler) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Done.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (tomicooler) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Rephrased it.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (tomicooler) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Removed and updated the flexible aqc part.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (tomicooler) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Done

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 29s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  markdownlint  |   0m  0s |  |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  33m 59s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 25s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  54m 51s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 13s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 16s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  20m 18s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 29s |  |  The patch does not generate ASF License warnings.  |
|  |   |  78m 58s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-5870/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/5870 |
| Optional Tests | dupname asflicense mvnsite codespell detsecrets markdownlint |
| uname | Linux cf217b818146 4.15.0-213-generic #224-Ubuntu SMP Mon Jun 19 13:30:12 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 2f58e5c3ca2bbdeba1808c0395b21ac110b34227 |
| Max. process+thread count | 725 (vs. ulimit of 5500) |
| modules | C: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site U: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-5870/2/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md
Comment (brumi1024) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md:309]: Yes, here I don't see an issue with it. If there are no other mentions of this maybe a short, few word explanation can help, mentioning that it might be already downscaled to fit in the parent's resources etc.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md
Comment (brumi1024) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md]: Sounds good to me.

**Review by brumi1024 — APPROVED** [review]
Thanks @tomicooler, the patch LGTM. One more question remains: since the RM REST doc is modified as well, we could either wait for YARN-11514 before merging this or we could handle it in a separate ticket. Either way is fine for me, what do you think?

**Comment by tomicooler** [comment]
@brumi1024 it can wait for YARN-11514, I used the base scheduler-response.json/xml from the tests.

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 28s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  markdownlint  |   0m  0s |  |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  36m 14s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 25s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  56m 57s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 13s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  mvnsite  |   0m 16s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  20m 31s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 29s |  |  The patch does not generate ASF License warnings.  |
|  |   |  82m  4s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-5870/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/5870 |
| Optional Tests | dupname asflicense mvnsite codespell detsecrets markdownlint |
| uname | Linux d131aa9fb98e 4.15.0-213-generic #224-Ubuntu SMP Mon Jun 19 13:30:12 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 6650c836b83a7284bf1032eb7812575d25f96dd2 |
| Max. process+thread count | 565 (vs. ulimit of 5500) |
| modules | C: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site U: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-5870/3/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by brumi1024** [comment]
Thanks @tomicooler for the patch, @slfan1989 for the review, merging to trunk.

**Pull request merged**
